assets/js/country_support.js

```diff
@@ -60,7 +60,13 @@ function loadCountrySupportTable(elem, fetch) {
   };
 
   fetch(`${idpBaseUrl || ''}/api/country-support?locale=${locale}`)
-    .then((response) => response.json())
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
```

Comment:
would `.json()` throw an error on its own? I'm wondering if we have the `.catch()` at the end why we need the guard?

Reply:
> would `.json()` throw an error on its own? I'm wondering if we have the `.catch()` at the end why we need the guard?

I, too, expected the `catch` to work on its own. Alas, it seems to fail without triggering the `catch`. I know `fetch` error handling is a bit tricky to work with. This solution was based on [StackOverflow advice](https://stackoverflow.com/questions/38235715/fetch-reject-promise-and-catch-the-error-if-status-is-not-ok).